### ISSUE TITLE
'h' and 'v' are erroneously highlighted in keybindings

### ISSUE THREAD

**Issue by Utagai** [mboughaba/i3config.vim#9]
Perhaps unsurprisingly, I like having my i3 follow vim-style movements. So this is a snippet from my config file:

```
# change focus
bindsym Mod1+h focus left
bindsym Mod1+j focus down
bindsym Mod1+k focus up
bindsym Mod1+l focus right
```

Since `v` and `h` are technically 'keywords' in i3 for splitting vertically and horizontally, the `Mod1+h` (and also potentially `Mod1+v`, or any other use of `v` without letters before it) line will get its `v` character highlighted.

This issue actually seems to exist in @PotatoesMaster's project [as well](https://github.com/PotatoesMaster/i3-vim-syntax/issues/12).

**Comment by mboughaba** [comment]
Hi @Utagai 

Thanks for posting the issue, I will take a look and comeback to you.

Cheers

**Comment by mboughaba** [comment]
Hi @Utagai 

I couldn't find the issue you are describing.

![screenshot from 2018-09-23 13-44-48](https://user-images.githubusercontent.com/3800584/45927684-0141dc00-bf37-11e8-92e0-d2e5a84eb256.png)

We are not doing anything special with `v` and `h` abbreviations. As you can see below.
https://github.com/mboughaba/i3config.vim/blob/50b8a21097b5b84719d5af620f448dd057091553/after/syntax/i3config.vim#L71
https://github.com/mboughaba/i3config.vim/blob/50b8a21097b5b84719d5af620f448dd057091553/after/syntax/i3config.vim#L84


Can you give more info and/or screenshots?

**Comment by Utagai** [comment]
@mboughaba It seems like this may have been (inadvertently?) fixed with any commits since the time this issue was posted. I don't see the issue I talked about here anymore.

It is also possible that I may have still been on @PotatoesMaster's branch and did not realize, so that is also worth noting. In either case, thanks!

I will say that I noticed something new with regards to highlighting numbers being too aggressive. I'd like to at least confirm that you see this as well before I file an issue for it. Here's a screenshot:

![number highlighting](https://i.imgur.com/XKSRrcn.png)

In particular, of interest is the `86`. This should be seen as part of the key name and therefore not highlighted.
